### PR TITLE
Fx background field color and refactor environment variables

### PR DESCRIPTION
**Pull request by juliaendre**
## [APP-XXX]

- Change 1
- Change 2

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift [hunk 1]
```diff
@@ -311,37 +311,27 @@ struct hFloatingTextField_Previews: PreviewProvider {
     }
 }
 
-private struct EnvironmentHWithoutDisabledColor: EnvironmentKey {
-    static let defaultValue = false
+public enum BackgroundColorType {
```
Comment (sladan-hedvig) [inline Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift:314]: Really good idea to handle them like this, but naming maybe needs to be updated like
`BackgroundOption` or something similar

**Review by sladan-hedvig — COMMENTED** [review]
Just make sure it works both on dark/light theme 😅

File: Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift (hunk 1 above)
Comment (juliaendre) [reply Projects/hCoreUI/Sources/hForm/hFloatingTextField.swift]: Yeah that sounds good! Updated it 👍

**Comment by juliaendre** [comment]
> Just make sure it works both on dark/light theme 😅

Checked it and I think it looks good :)

**Review by sladan-hedvig — APPROVED** [review]

**Pull request merged**